Client/Frontend/Browser/ReaderModeBarView.swift

```diff
@@ -109,17 +109,32 @@ class ReaderModeBarView: UIView {
         delegate?.readerModeBar(self, didSelectButton: added ? .RemoveFromReadingList : .AddToReadingList)
     }
 
+    private func updateUnread(unread: Bool) {
+        var buttonType: ReaderModeBarButtonType = unread ? .MarkAsRead : .MarkAsUnread
```

Comment:
Ha! No this is just confusing because of the way the images are named. If the article state is `unread` then we want to show the `MarkAsRead` button image.